mlrun/runtimes/pod.py

```diff
@@ -337,6 +341,27 @@ def with_node_selection(
         if affinity:
             self.spec.affinity = affinity
 
+    def with_priority_class(self, priority_class_name: typing.Optional[str]):
```

Comment:
```suggestion
    def with_priority_class(self, name: typing.Optional[str]):
```
I think name is good enough, `priority_class_name` is just too long and adds no information (it's clear that it's the name of the priority class from the name of the method)

Reply:
don't forget to update the docstring accordingly